vividus-plugin-web-app-to-rest-api/src/main/resources/org/vividus/steps/integration/redirect-validation-test-report.ftl

```diff
@@ -0,0 +1,76 @@
+<#ftl strip_whitespace=true>
+
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Redirect validation test report</title>
+    <link rel="stylesheet" href="../../styles.css"/>
+    <link rel="stylesheet" href="../../webjars/bootstrap/3.4.1/css/bootstrap.min.css"/>
+</head>
+<body>
+    <style>
+        .passed {
+            background-color: #DFF0D8;
```

Comment:
please avoid copying common styles, but rather reuse then

Reply:
> please avoid copying common styles, but rather reuse then

bootstrap 5 using another background color property for tables which is incompatible with bootstrap 3